app/models/user.rb

```diff
@@ -38,20 +38,21 @@ class User < ApplicationRecord
   has_many :authentication_tokens, dependent: :destroy
   has_many :spins, dependent: :destroy
 
-  validates :name,              presence: true
+  validates :id,                presence: true
+  validates :name,              presence: true, allow_blank: true
```

Comment:
The name should be present... How many people don't have name in github?

Reply:
That is exactly the error we found. There is the possibility of not having a name...